core/templates/dev/head/css/oppia.css

```diff
@@ -3393,8 +3528,53 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 4px 0 0 20px;
 }
 
-.oppia-lost-changes > li > .state-edit-desc > .sub-edit {
-  margin-left: 20px;
+.oppia-lost-changes > li > .state-edit-desc > .sub-edit {
+  margin-left: 20px;
+}
+
+.oppia-about-tabs li {
```

Comment:
Can we have this closer to the CSS for the main about pages? If not, why not?

In general, we should try and group related CSS. (And in the future perhaps move them altogether to their individual pages.)

Reply:
I think we cannot move this closer to the CSS for the main about pages. It might disturb the specificity of the whole page. But in the future, if we move them to their individual pages, we can easily do so.

Reply:
Ah sorry, is this referring to something on the About page or not? If not, it's fine here. If so, why does moving it disturb specificity? There shouldn't be any "might" or "might not" about this; we should understand the exact reasons.

Reply:
Yes, it refers to the tabs on the About page. I checked the specificity of `.oppia-about-tabs li` and `.oppia-info-card-content .card-metrics li` - turns out that the latter has a higher specificity than the former. (There's a tool to calculate specificity for CSS elements: [Specificity Calculator](https://specificity.keegan.st/)). Therefore we need to go with this order. We can arrange them when we separate individual pages though.